tests/unit-tests/scene/test_abstract_shell.cpp

```diff
@@ -93,6 +93,28 @@ struct MockSessionManager : ms::SessionManager
 {
     using ms::SessionManager::SessionManager;
 
+    // Must wrap the constructor as the NiceMock<> constructor can only take 10 arguments on older platforms
```

Comment:
It is amazing that a mock would require so many arguments.

Reply:
Indeed. The mock inherits from `scene::SessionManager`, and previously used its constructor directly. Unfortunately I added an 11th argument to the `scene::SessionManager` constructor, which blew up the mock on older gtest versions. This is the simplest fix. The ideal solution is probably to make the mock inherit from `scene::SessionCoordinator` (the interface) instead of `scene::SessionManager` (the implementation), but such a change is out of scope of this PR.